content/developer/reference/javascript/services.rst

```diff
@@ -113,13 +113,53 @@ Reference List
 
    * - Technical Name
      - Short Description
+   * - :ref:`cookie <services/cookie>`
+     - read or modify cookies
    * - :ref:`rpc <services/rpc>`
      - send requests to the server
    * - :ref:`title <services/title>`
      - read or modify the window title
    * - :ref:`user <services/user>`
      - provides some information related to the current user
 
+.. _services/cookie:
+
+Cookie service
+--------------
+
+Overview
+~~~~~~~~
+
+- Technical name: `cookie`
+- Dependencies: none
+
+Provides a way to manipulate cookies. For example:
+
+.. code-block:: javascript
+
+   cookieService.setCookie("hello", "odoo");
+
+API
+~~~
+
+.. js:data:: current
+
+   Object representing each cookie and its value if any (or empty string)
```

Comment:
`WARNING: duplicate data description of current, other data in developer/reference/javascript/services`

duplicate definition of same `js:data` is the cause of the red runbot.

Reply:
hmm, that's annoying. Not sure what to do here. I tried the `js:module::` directive (see 
https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#directive-js-module), the documentation says that it does not display a heading, which is what I want, but it does not work like that. It displays a ugly heading, just like classes. any idea?

Reply:
1) Either encapsulate it as a "false" class
![image](https://user-images.githubusercontent.com/19364378/139874081-405357ce-7646-4e2e-b283-c3e443115bb4.png)

2) Begin specifying methods and attributes with respect to their service:

![image](https://user-images.githubusercontent.com/19364378/139874136-5419b64b-d130-499f-b05c-7bc33b1a1290.png)

```suggestion
.. js:data:: cookieService.current

   Object representing each cookie and its value if any (or empty string)
```

I don't have any other solution keeping the js domain logic.

Reply:
i finallly chose to use the `:noindex:` flag, so it works as expected.